graphql/schema/types/performer.graphql

```diff
@@ -0,0 +1,196 @@
+enum GenderEnum {
+  MALE
+  FEMALE
+  TRANSGENDER_MALE
+  TRANSGENDER_FEMALE
+  INTERSEX
+}
+
+enum FakeTitsEnum {
+  YES
+  NO
+  NA
+}
+
+type Measurements {
+  cup_size: String
+  # TODO - should these be in cm to be consistent with height?
+  bust: Int
+  waist: Int
+  hip: Int
+}
+
+input MeasurementsInput {
+  cup_size: String
+  # TODO - should these be in cm to be consistent with height?
+  bust: Int
+  waist: Int
+  hip: Int
+}
+
+type Performer {
+  id: ID!
+  name: String!
+  disambiguation: String
+  aliases: [String!]!
+  gender: GenderEnum
+  urls: [URL!]!
+  birthdate: Date
+  ethnicity: String
+  country: String
+  eye_color: String
+  """Height in cm"""
+  height: Int
+  measurements: Measurements!
+  # TODO - is there a better desciptor for this - boob_job? - enhanced?
+  fake_tits: FakeTitsEnum
```

Comment:
I think `boob_job` is better, `fake_tits` sounds very negative. Could also use `breasts_augmented`, although that's maybe a bit verbose.

Reply:
Pornhub have "Breast type" with options "Natural" and "Fake". I like `breast_type`, for exactly the reason you describe.